assets/src/containers/ResourcesAccessed.js

```diff
@@ -213,21 +217,29 @@ function ResourcesAccessed (props) {
     }
   }
 
-  const ResourceAccessChartBuilder = (resourceData) => {
+  const resourceAccessChartBuilder = (resourceData, resourcesLimit) => {
     if (resourceTypeFilter.length === 0) {
       return (<AlertBanner>Please select a resource type to display data.</AlertBanner>)
     } else if (!resourceData || Object.keys(resourceData).length === 0) {
       return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
     } else {
       return (
-        <ResourceAccessChart
-          data={resourceData}
-          weekRange={weekRange}
-          gradeSelection={resourceGradeFilter}
-          resourceType={resourceTypeFilter}
-          aspectRatio={0.3}
-          minHeight={350}
-        />
+        <>
+          <div style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+            {typeof resourcesLimit === 'undefined' ? <div>Displaying all available resources in this course</div> :
+              <div>Displaying top {resourcesLimit} most accessed resources in this course</div>
+            }
+          </div>
+
```

Comment:
Nit-pick: We should probably get you set up to use `standard` linting. I think it would flag some of this.

Reply:
I had the extension installed but I checked it and looks like it needed a migration because the one I had was deprecated? I figured it wasn't reporting any problems so I was good. Updating and restarting I see it notices some issues. I can resolve that.

Reply:
Yeah, I think the `standard` VSCode plugin changed.